packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -49,6 +54,7 @@ def invoke_detect_ip():
     '-Djava.util.prefs.userRoot=/var/lib/dcos/exhibitor/',
     '-Duser.home=/var/lib/dcos/exhibitor/',
     '-Duser.dir=/var/lib/dcos/exhibitor/',
+    '-Djna.tmpdir=%s' % jna_tmpdir,
```

Comment:
Why do we need the `SERVER_JVMFLAGS` above if we're setting the value here? Should we just have a single environment variable for the value and then use it everywhere?